public/components/integrations/components/setup_integration.tsx

```diff
@@ -182,6 +184,38 @@ const runQuery = async (
   }
 };
 
+export function SetupWorkflowSelector({
```

Comment:
This is the main selector for the frontend, most of the other changes are wrappers around this